packages/calcite-components/src/components/table/table.e2e.ts

```diff
@@ -2034,7 +2034,7 @@ describe("keyboard navigation", () => {
         `#${rowHead.id}`,
         (el) => el.shadowRoot?.activeElement.shadowRoot?.querySelector("th").classList,
       ),
-    ).toEqual({ "0": CSS.numberCell });
+    ).toEqual({ "0": CSS.numberCell, "1": CSS.selectedCell });
```

Comment:
@macandcheese These tests needed to be updated due to [this change](https://github.com/Esri/calcite-design-system/pull/8594/files#diff-4cc3917081bc0192c2695ce4c6e2f58fb22b00f3b5856e6b9a0102754987248dL232). If this is not intentional, we can update `table-header` accordingly and revert 7d1f290a002a4da1081bc4436521feefcaecda9d.

Reply:
There were some incorrect merge resolutions that affected other items in Table as well, caught by Chromatic but auto-accepted... can you remove these test updates and I can work through the incorrect merge results?